src/BTConnectionManagerProxy.rep

```diff
@@ -31,11 +31,13 @@ class BTConnectionManagerProxy {
     SLOT(void disconnectDevice())
     SLOT(void setFakeTailMode(bool enableFakery))
     SLOT(void setDeviceName(const QString& deviceID, const QString& deviceName))
+    SLOT(void resetDeviceNames())
 
     // Designed to fill up a map with information, based on the command found in the QVariantMap key "command"'s value
     PROP(QVariantMap command)
     SLOT(QVariantMap getCommand(const QString& command))
 
     SIGNAL(messageReceived(const QString &sender, const QString &message))
     SIGNAL(message(const QString& message))
+    SIGNAL(deviceNamesResetted())
```

Comment:
Resetted isn't English, i'm afraid, this should be called deviceNameReset (yes, it's a lovely language like that). Though, also that isn't really what this does, rather the function clears the list, so it would make sense to call these two 
 (and the other places which have something similar) clearDeviceNames and deviceNamesCleared.

Reply:
Yup, sorry ;)